### DISCUSSION TITLE
Automatically post custom metric data

### DISCUSSION THREAD

**Discussion by rodrigogsqquid** [serverless/serverless#9853] [Q&A]
Hello!

Me and my team are trying to post data points for a custom Cloud Watch metric (AWS) from our lambda functions. Our approach so far is to create the metric in AWS and then use their SDK to post data points. 

Is there a way to do this automatically using the framework? 

Thanks!

**Comment by mnapoli** [comment]
Can you create the metric using CloudFormation (in the `resources` section of `serverless.yml`)?
